### DISCUSSION TITLE
Marking members of an organization that sponsors a person

### DISCUSSION THREAD

**Discussion by huonw** [community/community#9966] [Other Feature Feedback, Questions, & Ideas]
Currently it seems like an individual who sponsors has their sponsorships noted in a few places:

- on their profile
- as a badge when they interact with an repo that they sponsor (e.g. if I set up a sponsorship for `@example-user` via "Sponsor as @huonw", it seems there'll be a "Sponsor" badge when I file an issue or pull request against `example-user/example-repo`)

However, when an organisation sponsors, it seems like members of that organisation do not have this noted, even if they're acting on behalf of the organisation. For instance, my employer sponsors a bunch of developers and orgs via GitHub Sponsors, and I file work-related issues and PRs against those repos, but seemingly cannot have GitHub indicate that it's on behalf of the sponsorship.

This is especially relevant for the sponsorships that are large enough to meet a "priority response/support" threshold that some developers/orgs offer.

**Comment by jlord** [comment]
Thanks, this is great feedback and we're currently exploring sponsors recognition across the board and I'll bring this to our team.
